shared/src/business/entities/cases/PublicCase.test.js

```diff
@@ -94,6 +101,171 @@ describe('PublicCase', () => {
     });
   });
 
+  it('should show all contact and practitioner information if user has IRS Practitioner role', () => {
```

Comment:
Could these two tests be moved down to the `describe('irsPractitioner', () => {` block on line 4069?